### ISSUE TITLE
Loading schema in version 6/7 which refers schema with previous version (4)

### ISSUE THREAD

**Issue by MarcBouteiller** [everit-org/json-schema#164]
When a root schema is loaded with version 6 (or higher), with a nested reference with a version 4, the loader keeps the V6 spec version, and an exception is thrown because of the backwards incompatible changes of "exclusiveMinimum" and "exclusiveMaximum".

Example: 

```json
{
  "$schema": "http://json-schema.org/draft-06/schema#",
    "properties": {
      "parameter": {
        "type": "array",
          "items": {
             "$ref": "http://swagger.io/v2/schema.json#/definitions/parameter"
           }
        }
    }
}
```
The target of $ref is under version 4. #165

**Comment by erosb** [comment]
Right, that is an issue to be addressed. Thanks for opening an issue.
I would hugely appreciate if you could open a PR with a failing test for it, as it is [described in the contribution guidelines](https://github.com/everit-org/json-schema/blob/master/CONTRIBUTING.md).

Thanks

**Comment by erosb** [comment]
The fix is available in the latest release: [1.9.0](https://github.com/everit-org/json-schema/releases/tag/1.9.0)
